<ui version="4.0" >
 <class>CutSet</class>
 <widget class="QWidget" name="CutSet" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>681</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title" >
    <string>Fileset to be cut</string>
   </property>
   <widget class="QTableWidget" name="tableWidgetFileSets1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>661</width>
      <height>116</height>
     </rect>
    </property>
    <property name="autoFillBackground" >
     <bool>true</bool>
    </property>
    <property name="selectionBehavior" >
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
    <column>
     <property name="text" >
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Projection</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Resolution</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>File Format</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Use</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Cutted Set</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Classification</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayout" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>656</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>File Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxFileType1" >
       <item>
        <property name="text" >
         <string>DEM</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Clutter</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>File Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxFileFormat1" >
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <item>
        <property name="text" >
         <string>All</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>GDALFILE</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>ORTFILE</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Projection:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxProjection1" >
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <item>
        <property name="text" >
         <string>All</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>DEG</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>WGS84GC</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>UTM</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>170</y>
     <width>681</width>
     <height>136</height>
    </rect>
   </property>
   <property name="title" >
    <string>Fileset containing cut data</string>
   </property>
   <widget class="QTableWidget" name="tableWidgetFileSets2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>661</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground" >
     <bool>true</bool>
    </property>
    <property name="selectionBehavior" >
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
    <column>
     <property name="text" >
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Projection</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Resolution</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>File Format</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Use</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Cutted Set</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Classification</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayout_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>310</y>
     <width>661</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Save cut files in directory:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDirectory" />
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonBrowse1" >
      <property name="text" >
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCut1" >
      <property name="text" >
       <string>Cut</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
